qcfractal/cli/qcfractal_server.py

```diff
@@ -481,6 +486,53 @@ def server_dashboard(args, config):
     app.run_server(debug=True)
 
 
+def server_backup(args, config):
+    psql = standard_command_startup("backup", config)
+
+    print("\n>>> Starting backup, this may take several hours for large databases (100+ GB)...")
```

Comment:
Might be helpful to print the user's database size here.

Reply:
Can do.